docs/products/clickhouse/howto/data-service-integration.md

```diff
@@ -1,134 +1,240 @@
 
-This article details how to set up and use data service integrations in
-Aiven for ClickHouse.
-
-For information on how to set up and use regular integrations in Aiven
-for ClickHouse, see intregration guides in
-[Integrate your Aiven for ClickHouse® service](/docs/products/clickhouse/howto/list-integrations).
-
-## About data service integrations
-
-By enabling data service integrations in Aiven for ClickHouse, you
-create streaming data pipelines across services. Aiven for ClickHouse
-supports data service integrations with Aiven for Kafka® and Aiven for
-PostgreSQL®.
-
-You can create Aiven for ClickHouse® data service integrations in the
-[Aiven web console](https://console.aiven.io/).
-
-:::note[Integration databases]
-When creating integrations in the **Data service integrations** wizard,
-you can also create integration databases connected to the services you
-are integrating with.
-
-If you prefer to create a data service integration without adding
-integration databases, you can create integration databases for your
-service any time later. See
-[Manage Aiven for ClickHouse® integration databases](/docs/products/clickhouse/howto/integration-databases) for guidance on how to do that.
+:::tip
+Read about data service integration types and methods available with Aiven for
+ClickHouse® in
+[Aiven for ClickHouse® data service integration](/docs/products/clickhouse/concepts/data-integration-overview).
 :::
 
 ## Prerequisites
 
--   Aiven account
--   Access to [Aiven web console](https://console.aiven.io/)
+- You are familiar with the limitations listed in
+  [About Aiven for ClickHouse® data service integration](/docs/products/clickhouse/concepts/data-integration-overview#supported-data-source-types).
+- You have an organization, a project, and an Aiven for ClickHouse service in Aiven.
+- You have access to the [Aiven Console](https://console.aiven.io/).
 
 ## Create data service integrations {#create-data-service-integration}
 
-1.  Log in to the [Aiven web console](https://console.aiven.io/).
-
-1.  In the **Services** page, select an Aiven for ClickHouse service you
-    want to integrate with a data service.
-
-1.  Select **Get started** from the **Integrate your Aiven for
-    ClickHouse** section in the **Overview** page of your service.
-
-1.  In the **Data service integrations** wizard, select one of the
-    following options:
-
-    **Option 1: Create a service and integrate it**
+1. Log in to the [Aiven Console](https://console.aiven.io/), and go to an organization
+   and a project.
+1. From <ConsoleLabel name="services"/>, select an Aiven for ClickHouse service to integrate
+   with a data source.
+1. In the service <ConsoleLabel name="overview"/>, go to **Data pipeline** and click one
+   of the following:
+
+   - **Add data source** if you're setting up the first integration for that service
+   - <ConsoleIcon name="plus"/> if you're setting up another integrtion for that service
```

Comment:
Perhaps it would read better with an action in the substeps? Something like this maybe?

```suggestion
1. In the service <ConsoleLabel name="overview"/>, go to **Data pipeline**:

   - To create your first integration, click **Add data source**.
   - To add an integration, click <ConsoleIcon name="plus"/>.
```